flink-core/src/main/java/org/apache/flink/configuration/ConfigOption.java

```diff
@@ -137,6 +153,28 @@ public boolean hasDeprecatedKeys() {
 		return deprecatedKeys == EMPTY ? Collections.<String>emptyList() : Arrays.asList(deprecatedKeys);
 	}
 
+	public String shortDescription() {
+		return shortDescription;
+	}
+
+	public String description() {
+		return description;
+	}
+
+	String toHTMLString(boolean includeShort) {
```

Comment:
Can we move this method to `ConfigGroup` as well? All accessed fields are available from the outside as well.